tests/components/nextdns/test_config_flow.py

```diff
@@ -60,7 +61,7 @@ async def test_form_create_entry(hass):
         (ValueError, "unknown"),
     ],
 )
-async def test_form_errors(hass, error):
+async def test_form_errors(hass: HomeAssistant, error: tuple[Exception, str]) -> None:
```

Comment:
Split the `error` parameter already into two parameters, eg `"exc, base_error"` instead.